code/modules/food_and_drinks/recipes/tablecraft/recipes_drink.dm

```diff
@@ -114,3 +114,14 @@
 	time = 10
 	reqs = list(/obj/item/stack/sheet/cardboard = 1)
 	category = CAT_DRINK
+
+/datum/crafting_recipe/candycornwhiskey
+	name = "candy corn whiskey"
+	result = /obj/item/reagent_containers/food/drinks/bottle/candycornwhiskey
+	time = 30
+	reqs = list(/datum/reagent/consumable/ethanol/whiskey = 5,
+				/obj/item/reagent_containers/food/snacks/candy_corn = 1,
+				/obj/item/reagent_containers/food/drinks/bottle = 1)
+	category = CAT_DRINK

```

Comment:
so you're not gonna do my kong crafting recipe suggestion?

Reply:
Not unless there's an object or reagent that plausibly explains its literal depressant effects. I haven't thought too much about it. Having it be whiskey and a monkey cube is sort of like my original idea of making the reagent gorillise you on stage 4 withdrawal which isn't consistent with the reference. Also reminds me of when upgrading security cameras for x-ray required a [carrot](https://www.ss13.eu/wiki/index.php/Guide_to_construction#Security_Camera).

Reply:
The joke is carrots improve your eyesight.

Reply:
I know. 
I changed my mind about this.